Subject.cs

```diff
@@ -0,0 +1,14 @@
+namespace assignment_3;
+
+public class Subject
+{
+    public int SubjectId { get; set; }
+    public string SubjectName { get; set; }
+    public int[] GradingScale { get; set; } = new int[] {1,2,3,4,5};
+
+    public void UpdateSubjectName(string newSubjectName)
```

Comment:
This method is redundant since

`public string SubjectName { get; set; }`

already has a public setter. Not a huge deal now, but something to think about later.